lib/internal/test_runner/mock/mock_timers.js

```diff
@@ -64,10 +65,12 @@ class MockTimers {
   #realTimersSetInterval;
   #realTimersClearInterval;
 
+  #realDate;
+
   #timersInContext = [];
   #isEnabled = false;
   #currentTimer = 1;
-  #now = DateNow();
+  #now = 0;
```

Comment:
#now should be DateNow unless it's changed by .setTime

Reply:
But that differs from the implementation from `fake-timers`. If the intention is to be close to that [API](https://sinonjs.org/releases/v15/fake-timers/) then we wouldn't be able to start it as `DateNow`, however, I'm not against that too. I think it's worth a discussion.

On my side I think the pros is that you can start the Date object without actually setting anything on it, as opposed to having to set the time at any given new fake timer instance because otherwise you'd have Jan 1, 1970. Which also can be a pro to some people as it's more explicit to what time is the initial time.